app.js

```diff
@@ -2,17 +2,63 @@ require('dotenv').config();
 
 const express = require('express');
 const hbs = require('hbs');
-
-// require spotify-web-api-node package here:
-
+const SpotifyWebApi = require('spotify-web-api-node');
 const app = express();
 
 app.set('view engine', 'hbs');
 app.set('views', __dirname + '/views');
 app.use(express.static(__dirname + '/public'));
 
 // setting the spotify-api goes here:
+const spotifyApi = new SpotifyWebApi({
+  clientId: process.env.CLIENT_ID,
+  clientSecret: process.env.CLIENT_SECRET,
+});
+
+// Retrieve an access token
+spotifyApi
+  .clientCredentialsGrant()
+  .then(data => spotifyApi.setAccessToken(data.body['access_token']))
+  .catch(error => console.log('Something went wrong when retrieving an access token', error));
 
 // Our routes go here:
+app.get('/', (req, res) => {
+  res.render('homepage')
+})
+
+app.get('/artist-search', (req, res) => {
+  spotifyApi
+  .searchArtists(req.query.artist)
```

Comment:
A good practice and for easy to read its to save the querys in variables first and then pass it something like
```
const searchArtist =req.query.artist
  spotifyApi
    .searchArtists( searchArtist)
    ```